building_sim_plugins/building_plugins_common/src/crowd_simulator_common.cpp

```diff
@@ -226,7 +223,7 @@ void CrowdSimInterface::_add_object(AgentPtr agent_ptr,
 
 size_t CrowdSimInterface::get_num_objects() const
 {
-  return _objects.size();
+  return (_menge_disabled) ? _internal_agents.size() : _objects.size();
```

Comment:
Is there a difference on the number of objects in simulation whether menge is enabled or not?

Reply:
When menge is disabled, `init_crowd_sim` is not called and hence `_objects` are not loaded. The workaround is to use _internal_agents (just the names read from sdf for idle animation).